proposed/0000-Use-public-key-for-id-on-all-messages.md

```diff
@@ -0,0 +1,60 @@
+- Feature Name: Use public key alone to authorise actions 
+- Type Enhancement
+- Related components routing, maidsafe_types, maidsafe_client, maidsafe_vaults
+- Start Date: 23-06-2015
+- RFC PR: 
+- Issue number: 
+
+# Summary
+
+At the moment client Id packets are create and a hash of these packets used as identity. 
+This involves storing such packets prior to the client interacting with the network. 
+As such this requires an initial `unauthorised put` as the client is not known to the 
+network and cannot be recognised without this. It also means there are many lookups
+to link the hash of the id packet with the client taking the action. 
+These lookups are costly it is proposed here that they are not required.  
+
```

Comment:
I think, it should be elaborated on what is the reasoning behind the current mechanism and how the mechanism works (either here or providing a link to documentation). Also a formal definition of `action` would be handy somewhere. Otherwise I think it's not a RFC, but just a memo for those who already know the context.